R/build-news.R

```diff
@@ -147,6 +147,8 @@ data_news <- function(pkg = ".") {
 
   sections <- xml2::xml_find_all(xml, "./body/div")
 
+  if (pkg$bs_version > 3) tweak_section_levels(xml)
```

Comment:
```suggestion
  if (pkg$bs_version > 3) {
    tweak_section_levels(xml)
  }
```

I think our style guide isn't quite right; we always use `{}` in situations like this.